### ISSUE TITLE
runspv/glsl-to-spv-worker: glslangValidator, spirv-opt, etc. can fail. Handle this somewhere.

### ISSUE THREAD

**Issue by hevrard** [google/graphicsfuzz#301]
Right now I can see this in the worker trace:

```
No job
#### Image job: reference
test.vert
reference.frag
ERROR: reference.frag:3: '' :  syntax error, unexpected IDENTIFIER
ERROR: 1 compilation errors.  No code generated.


ERROR: Linking fragment stage: Missing entry point: Each stage requires one entry point

SPIR-V is not generated for failed compile or link
Send back, results status: 70
No job
```

Yet the log passed on to the server is:
```
Start: reference
ERROR
COMMAND: /usr/local/google/home/hevrard/work/graphicsfuzz/graphicsfuzz/target/graphicsfuzz/python/drivers/../../bin/Linux/glslangValidator reference.frag -V -o reference.frag.spv
RETURNCODE: 2
STDOUT: None
STDERR: None
```

**Comment by paulthomson** [comment]
I have fixed this for runspv, but I need to fix up glsl-to-spv-worker. I.e. Issue #300

**Comment by paulthomson** [comment]
Fixed in #354.
